internal/cli/clusters/describe.go

```diff
@@ -49,12 +55,31 @@ var describeTemplate = `ID	NAME	MDB VER	STATE
 func (opts *DescribeOpts) Run() error {
 	r, err := opts.store.AtlasCluster(opts.ConfigProjectID(), opts.name)
 	if err != nil {
+		return opts.RunFlexCluster(err)
+	}
+
+	return opts.Print(r)
+}
+
+func (opts *DescribeOpts) RunFlexCluster(err error) error {
+	apiError, ok := atlasClustersPinned.AsError(err)
+	if !ok {
 		return err
 	}
+
+	if *apiError.Error != cannotUseFlexWithClusterApisHTTPCode && *apiError.ErrorCode != cannotUseFlexWithClusterApisErrorCode {
```

Comment:
Afaik it's safe to only rely on the `ErrorCode`

Reply:
I will remove it